.github/workflows/main.yml

```diff
@@ -35,28 +35,46 @@ jobs:
           - "3.4"
           - ruby-head
           - jruby-9.4
-        task:
-          - internal_investigation
-          - spec
-    name: "Ruby ${{ matrix.ruby }}: ${{ matrix.task }}"
+        parser_engine:
+          - parser_whitequark
+        include:
+          - ruby: "2.7" # Minimum version required for Prism to run.
+            parser_engine: parser_prism
```

Comment:
https://github.com/rubocop/rubocop/blob/d8448a3790a78f16536e267ed7ea5ffc2e03561b/docs/modules/ROOT/pages/compatibility.adoc#L44

> Starting from RuboCop 1.75, the prism translation layer is used by default when analyzing Ruby 3.4+.

What is the point for us to run this job on 2.7, and tell the Prism to analyze the code as it was Ruby 3.4? Don't we have those 3.4-specific (eg it-block) specs disabled when running on Ruby 2.7?

Is it just my OCD, or we need two Prism jobs, one on 2.7, to ensure it runs on older rubies, and one on 3.4, to ensure cops do what we expect them to, including new features, with Prism?